scripts/bench/utils.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line depend/ban-dependencies
```

Comment:
perhaps we should instead just scope the rule to the `code` dir?

I'm afraid having too many eslint disable lines will just make that the pattern what people use. [The Broken windows theory](https://en.wikipedia.org/wiki/Broken_windows_theory).

Reply:
We'd flag those newly added ignore lines in PR reviews.